packages/typespec-ts/src/modular/buildOperations.ts

```diff
@@ -11,6 +11,7 @@ import { Client, ModularCodeModel, Operation } from "./modularCodeModel.js";
 import { isRLCMultiEndpoint } from "../utils/clientUtils.js";
 import { getDocsFromDescription } from "./helpers/docsHelpers.js";
 import { SdkContext } from "../utils/interfaces.js";
+import { getImportSpecifier } from "@azure-tools/rlc-common";
```

Comment:
It strikes me as odd to see an rlc-common import in the modular codegen layer. Probably okay but wanted to bring this up in case there is an alternative

Reply:
Good catch up! Let me update this to modular.